static/js/ConnectionsPanelHeader.jsx

```diff
@@ -117,10 +117,14 @@ class ConnectionsPanelHeader extends Component {
       const toggleLang = Sefaria.util.getUrlVars()["lang2"] === "en" ? "he" : "en";
       const langUrl = Sefaria.util.replaceUrlParam("lang2", toggleLang);
       const closeUrl = Sefaria.util.removeUrlParam("with");
-      const showOneLanguage = !Sefaria._siteSettings.TORAH_SPECIFIC || Sefaria.interfaceLang === "hebrew";
-      const toggleButton =  (showOneLanguage) ? null : (this.props.connectionsMode === 'TextList') ?
-          <DropdownMenu buttonContent={<DisplaySettingsButton/>} context={ReaderPanelContext}><ReaderDisplayOptionsMenu/></DropdownMenu> :
-            <LanguageToggleButton toggleLanguage={this.props.toggleLanguage} url={langUrl} />;
+      const showOneLanguage = !Sefaria._siteSettings.TORAH_SPECIFIC;
+      const isEnglishUI = this.props.interfaceLang === "english";
+      const toggleButton = (showOneLanguage)
```

Comment:
Flip the if-statement to be `this.props.connectionsMode === "Resources". Pull out this logic into its own function for clarity. Function can be called something like "getLanguageSwitcher()"